server/resources/migrations/49_add_app_upload_urls.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE app_upload_urls (
```

Comment:
Can you add an index on `app_upload_urls (app_id)`. That way we don't have to do a full table scan when an app is deleted.